test/spec/proxySpec.js

```diff
@@ -1148,6 +1146,86 @@ describe('proxy', function() {
       // }, 100);
     });
   });
+  describe('Already proxified values', function() {
+    it("shouldn't re-proxify (it should deep clone) proxified values from different instances", function() {
```

Comment:
If that's the nature of proxies then I think we have to live with that. On what occasions "our developers" can run into this? 

If they write views that conform to the convention, then they will never use `proxifiedObject` directly, will they?